spec/integration/workbook/images.spec.js

```diff
@@ -218,7 +218,7 @@ describe('Workbook', () => {
           expect(images.length).to.equal(1);
 
           const imageDesc = images[0];
-          expect(imageDesc.range.editAs).to.equal('oneCell');
+          expect(imageDesc.range.editAs).to.equal('absolute');
```

Comment:
@alubbe  if not fixed, the actual value will be `oneCell`.
Since there are already many tests for `oneCell`, I am directly modifying my previous test cases.

Reply:
ok, this should cover these changes :)

have we any test for "oneCell" that makes exactly same thing like this one?

Reply:
> have we any test for "oneCell" that makes exactly same thing like this one?

ok, it is:)